pandas/tests/test_frame.py

```diff
@@ -5545,6 +5545,8 @@ def test_itertuples(self):
         dfaa = df[['a', 'a']]
         self.assertEqual(list(dfaa.itertuples()), [(0, 1, 1), (1, 2, 2), (2, 3, 3)])
 
+        self.assertEqual(type(next(df.itertuples(name=None))), tuple)
```

Comment:
@maximilianr: Thanks for the hint, `pandas.common.core.is_named_tuple()` uses `hasattr(..., '_fields')`. Is that to be prefered?

@kawochen: See above, the test (as I understand it) is to check the returned values are plain tuples, but `namedtuple('spam', 'spam eggs')(1, 2) == (1, 2)`.